iocore/net/SSLNetVConnection.cc

```diff
@@ -31,9 +31,13 @@
 #include "BIO_fastopen.h"
 #include "Log.h"
 #include "P_SSLClientUtils.h"
+#include "P_SSLSNI.h"
+#include "HttpTunnel.h"
 
+#include <unordered_set>
```

Comment:
This will raise strong objections - can you try to use `HashMap` from `lib/ts/Map.h`?

Reply:
alright. I ported this from my 5.3 patch. Forgot to change this